src/Process/Pool/Logger.php

```diff
@@ -49,6 +54,15 @@ public function log($level, $message, array $context = [])
                 $logMessage->setProcessId($processId);
                 $logMessage->setProcessPath($this->_getProcess()->getPath());
                 $logMessage->setMessage($message);
+
+                if (array_key_exists(self::CONTEXT_KEY_EXCEPTION, $context) && $context[self::CONTEXT_KEY_EXCEPTION]
+                instanceof \Throwable){
+                    $context[self::CONTEXT_KEY_EXCEPTION_STRING] = (string)$context[self::CONTEXT_KEY_EXCEPTION];
+                    $normalizedException = (new NormalizerFormatter())->format([$context[self::CONTEXT_KEY_EXCEPTION]]);
+                    unset($context[self::CONTEXT_KEY_EXCEPTION]);
+                    $context[self::CONTEXT_KEY_EXCEPTION] = $normalizedException[0];
```

Comment:
Is this offset always guaranteed? Is there any risk of an error or exception with `normalizedException` or the use of `NormalizerFormatter`? 

I think that's the only question I have to be gt🐐

Reply:
@rhift @mucha55 
The offset is guaranteed since the input was passed as an array (as required)  containing a single object that has already been verified as a `\Throwable`. The `(new NormalizerFormatter())->format([$throwable])` method will always (though not strictly typed) return an array with the normalized version of the `\Throwable` that we gave it.

The only exception that can be thrown by this method is if `json_encode` isn't available, and if thats the case we have other logging problems: 
```
if (!function_exists('json_encode')) {
            throw new \RuntimeException('PHP\'s json extension is required to use Monolog\'s NormalizerFormatter');
        }
```

Reply:
> As of PHP 5.2.0, the JSON extension is bundled and compiled into PHP by default

https://www.php.net/manual/en/json.installation.php

I guess we could add `ext-json` to `composer.json`, but 👍